script.js

```diff
@@ -1 +1,15 @@
-console.log("Hello World!");
\ No newline at end of file
+const form = document.getElementById("todoForm");
+const input = document.getElementById("inputField");
+const list = document.querySelector(".todos-list");
```

Comment:
I'd prefer using `document.getElementById()` in order to make sure that the particular `ul` is selected. selection using `class` is more suitable for styling inside javascript.